src/main/java/com/example/TournamentProject/TournamentManagement.java

```diff
@@ -0,0 +1,77 @@
+package com.example.TournamentProject;
+
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class TournamentManagement {
+
+    public void getTournamentName() {
+        System.out.println("Enter tournament name:");
+    }
+
+    public String createTournamentName() {
+        String tournamentName = null;
+        try {
+            BufferedReader buffer = new BufferedReader(new InputStreamReader(System.in));
+            tournamentName = buffer.readLine();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return tournamentName;
+    }
+
+    public Team pickWinner(Team home, Team guest) {
+
+        int homeInt = 0;
+        int guestInt = 0;
+        String homeString;
+        String guestString;
+        List<Team> nextRoundList = new ArrayList<>();
```

Comment:
why are you using here List if you add only 1 element ?

```
           if (homeInt > guestInt) {
                nextRoundList.add(home);
            } else nextRoundList.add(guest);
```
Looks like Collection is not the best choise for holding SINGLE element